cors.go

```diff
@@ -89,11 +204,27 @@ func AllowedMethods(methods []string) CORSOption {
 // Note: Passing in a []string{"*"} will allow any domain.
 func AllowedOrigins(origins []string) CORSOption {
 	return func(ch *cors) error {
+		for _, v := range origins {
+			if v == corsOriginMatchAll {
+				ch.allowedOrigins = []string{corsOriginMatchAll}
+				return nil
+			}
+		}
+
 		ch.allowedOrigins = origins
 		return nil
 	}
 }
 
+// ExposeHeaders are additional headers outside of those which are apart
+// of the simple response headers (http://www.w3.org/TR/cors/#simple-response-header)
+func ExposedHeaders(headers []string) CORSOption {
+	return func(ch *cors) error {
+		ch.exposedHeaders = headers
+		return nil
+	}
+}
+
 // MaxAge determines the maximum age (in seconds) between preflight requests. A
 // maximum of 10 minutes is allowed. An age above this value will default to 10
```

Comment:
You can probably fix my quick & dirty docs here on your next pass and say "... is allowed as per the CORS specification." to make it clearer to package users :100: